packages/esm-appointments-app/src/calendar/components/Cell/WeeklyCell.tsx

```diff
@@ -0,0 +1,27 @@
+import { Dayjs } from 'dayjs';
+import { CalendarType } from '../../utils/types';
+import { isSameMonth } from '../../functions/monthly';
+import styles from './WeeklyCell.module.scss';
+import React from 'react';
+
+function WeeklyCell({ type, dateTime, currentDate }: { type: CalendarType; dateTime: Dayjs; currentDate: Dayjs }) {
+  return (
+    <div
+      className={
+        styles[
+          type === 'weekly'
+            ? 'weekly-cell'
+            : isSameMonth(dateTime, currentDate)
+            ? 'monthly-cell'
+            : 'monthly-cell-disabled'
```

Comment:
Just a minor comment, I wouldn't expect the "WeeklyCell" component handling cases for "MonthlyCell".